src/export_blob_metrics.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+import logging
+from pathlib import Path
+import pandas as pd
+from omegaconf import DictConfig
+from tqdm import tqdm
+import os
+from utils.utils import read_yaml
+
+log = logging.getLogger(__name__)
```

Comment:
let's try and use logging throughout the script for debugging and general monitoring. See my soon-to-come commit